src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -8,16 +8,16 @@ namespace Microsoft.CodeAnalysis.CSharp.CodeStyle
     internal static class CSharpCodeStyleOptions
     {
         // TODO: get sign off on public api changes.
-        public static readonly Option<bool> UseVarWhenDeclaringLocals = new Option<bool>(nameof(CSharpCodeStyleOptions), nameof(UseVarWhenDeclaringLocals), defaultValue: true,
+        public static readonly Option<bool> UseVarWhenDeclaringLocals = new Option<bool>(nameof(CodeStyleOptions), nameof(UseVarWhenDeclaringLocals), defaultValue: true,
```

Comment:
I opened VS with the Feature name as `CSharpCodeStyleOptions` and set all the options to Error. I closed VS, changed the Feature name to `CodeStyleOptions`, ran VS again, and they were still all Errors. I then changed them all to Suggestions, switched back to `CSharpCodeStyleOptions`, reran VS, and they were all still Suggestions. 

This leads me to believe that there won't be a version-to-version upgrade issue around these options due to this change of Feature name, but I feel like I _might_ be missing something subtle here. @jasonmalinowski (or others) - Do you see any problem with changing the Feature name of these options?